src/richie/apps/courses/templates/courses/plugins/course_plugin.html

```diff
@@ -0,0 +1,12 @@
+{% load cms_tags i18n %}
+{% spaceless %}
+{% if current_page.publisher_is_draft or instance.check_publication is True %}
+  <div class="course-plugin-container {% if relevant_page.publisher_is_draft %} course-plugin-container--draft{% endif %}">
+    <div class="course-plugin">
+      <a class="course-plugin__body course-plugin__link" href="{{ relevant_page.get_absolute_url }}" title="{{ relevant_page.get_title }}" alt="{{ relevant_page.get_title }} {% trans 'logo' %}">
```

Comment:
I don't think you should have both `__body` and `__link` on the same BEM element name.

If this is a `course-glimpse` and a `course-glimpse--link` just put both on the same class and use it here :)

Reply:
should be better now 👍